modules/end-user-guide/partials/proc_using-private-container-registries.adoc

```diff
@@ -0,0 +1,37 @@
+[id="proc_using-private-container-registries_{context}"]
+= Using private container registries
+
+[role="_abstract"]
+This section describes the necessary steps to use container images from private container registries.
+
+.Prerequisites
+
+* A running instance of {prod}. To install an instance of {prod}, see xref:installation-guide:installing-che.adoc[].
+
+.Procedure
+
+. Navigate to the {prod-short} Dashboard. See xref:navigating-che-using-the-dashboard.adoc[].
+
+. Navigate to *User Preferences*.
+
+.. Click on your username in the top right corner.
+
+.. Click the btn:[User Preferences] tab.
+
+. Click the btn:[Add Container Registry] button in btn:[Container Registries] tab and execute following actions:
+
+** Enter the container registry domain name in the *Registry* field.
```

Comment:
ASAIS most registries (2 of 3 =) ) has domain as registry name, but docker.io has the full URL after I did docker login. I wonder if docker.io will work as well.

```json
> cat ~/.docker/config.json
{
        "auths": {
                "https://index.docker.io/v1/": {
                        "auth": "..."
                },
                "quay.io": {
                        "auth": "..."
                },
                "registry.redhat.io": {
                        "auth": "..."
                }
        }
}
```
Also, it may make sense to add reference to pull secrets https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/, but I'm not sure.

Reply:
Yes, I am not sure how to communicate that clearly without causing confusion. If anyone has a suggestion about the correct phrasing I can adjust it.

About the reference: I can create a *Additional resources* section and put that there.

Reply:
@xbaran4  maybe add a small note right under this step with clarification?
Smth like "For docker.io, the registry name is the full URL. For other registries, the registry name is usually a domain name".

+1 to additional resources

Reply:
> Smth like "For docker.io, the registry name is the full URL. For other registries, the registry name is usually a domain name".

docker.io may work as well. I believe @xbaran4 tested it during the new Dashboard Click party.

Reply:
![image](https://user-images.githubusercontent.com/73115616/112518409-a1dc1580-8d99-11eb-8916-97703dd4f941.png)
![image](https://user-images.githubusercontent.com/73115616/112518655-dbad1c00-8d99-11eb-9c9f-5fb6f44460ef.png)
I tried this setup and in devfile specified image: 'docker.io/pbaran555/che-cpp-rhel7:test' and pulled successfully.
So I believe we dont have adjust it.